react-native-sdk/index.tsx

```diff
@@ -74,22 +75,23 @@ export const JitsiMeeting = forwardRef(({ flags, meetingOptions, style }: IAppPr
 
     useEffect(
         () => {
-            const urlJwt = meetingOptions.url.jwt ? `?jwt=${meetingOptions.url.jwt}` : '';
-            const url = `${meetingOptions.url.domain}/${meetingOptions.url.roomName}${urlJwt}`;
+            const urlJwt = meetingOptions.url.token ? `?jwt=${meetingOptions.url.token}` : '';
```

Comment:
Why do we need this? The token is added to the URL automatically in the RN side, we just need to pass it around.